### ISSUE TITLE
Unknown type name 'RCTBubblingEventBlock'

### ISSUE THREAD

**Issue by pankajnegi-soar187** [react-native-community/lottie-react-native#444]
### Description
When I have try pod install after that when I open Xcode I getting Unknown type name 'RCTBubblingEventBlock' Error.
[Description of the bug]
My Pod File:-

platform :ios, '9.0'

target 'MyApp' do
  
  #use_frameworks!
  pod 'ReactNativePermissions', :path => '../node_modules/react-native-permissions'
  pod 'lottie-ios', :path => '../node_modules/lottie-ios'

  pod 'lottie-react-native', :path => '../node_modules/lottie-react-native'

end

### Versions
Install Node (version:- v11.8.0)
npm (version :- 6.5.0)
react-native-cli@2.0.1

**Comment by seruybrat** [comment]
I have the same issue

**Comment by li-yuan2** [comment]
@pankajnegi-soar187 @seruybrat
I didn't install lottie-react-native by cocoapods, so I'm not quite sure. But I noticed that your pod file didn't use local react-native file, that may be the problem.
lottie-react-native depends on React project, if you didnt't install React in pod file explicitly, cocoapods will install the React from default cocoapods repository, which has beed deprecated by react-native team. So after pod install, you may see a yellow warning "[!] React has been deprecated".
In that deprecated React version (0.11.0), the type RCTBubblingEventBlock has not been defined yet.
You can using your local react-native instead of that deprecated version, as described in the following comment:
https://github.com/react-native-community/react-native-camera/issues/1747#issuecomment-439277989
I haven't tested it perfectly, so I can't promise anything. Just some information for you :)

**Comment by pankajnegi-soar187** [comment]
@li-yuan2 may be you are right, becuase "[!] React has been deprecated" xcode showing error. But I resolved this error by removing below two lines form Pod file:-

pod 'lottie-ios', :path => '../node_modules/lottie-ios'

pod 'lottie-react-native', :path => '../node_modules/lottie-react-native'

After this I used pod install. and its work for me :) . and also lotti animation is working fine.
